src/components/DetailDrawer.tsx

```diff
@@ -195,6 +199,11 @@ export const DetailDrawer: React.FC<{
       <TableRowCustom>
         {columns.map(column => {
           if (shownColumns.includes(column.rowValueDescriptor)) {
+            const unformattedValue =
+              field[column.rowValueDescriptor as keyof ILookmlModelExploreField]
+            const displayValue = unformattedValue
```

Comment:
`String(undefined)` returns `null` which is displayed to screen. Passing in `undefined` to the formatter creates a `<td />` with no children, as expected